index.html

```diff
@@ -77,67 +78,76 @@ <h2>Examples of Usage</h2>
         </ol>
-        <p class="note">The overall goal for this api is to allow user agents to return true if they think an event may end up getting dispatched to a browsing context soon. Not only if an event is already placed into the queue. This may mean that a user agent may look into the current configuration of iframes, other events in the queue, or other information when making decisions in step 5b. For example, there may be cases where if a mousedown event is queued then a click event may be fired for this frame after the mousedown event. However after the mousedown event is fired the user agent knows that the click event will not be fired. In this case the user agent should return 'true' for a 'click' input before the mousedown event, but after it knows that click will not be fired it should return false.</p>
+        <p class="note">The overall goal for this api is to allow user agents to return true if they think an event may end up getting dispatched to a browsing context soon. Not only if an event is already placed into the queue. This may mean that a user agent may look into the current configuration of iframes, other events in the queue, or other information when making decisions in step 4.1. For example, there may be cases where if a mousedown event is dispatched then a click event may be fired for this frame after the mousedown event. However if the mousedown event is canceled then the user agent knows that the click event will not be fired. In this case the user agent should return 'true' for a 'click' input before the cancling mousedown event, but after it knows that a click will not be fired it should return false.</p>
 
-        <p class="note">The specification for the isInputPending method is intentionally written in such a way that a user agent may return true or false for any reason. This is so user agents may implement their own security measures while implementing this api and remain compliant with this specification.</p>
-      </section>
-      <section>
-        <h3>The <dfn data-link-for="canCheckInputPending">canCheckInputPending</dfn> method</h3>
-        <p>The <code>canCheckInputPending</code> method returns a value indicating if this api is supported in this environment and optionally if a specific InputType is supported. If the <code>canCheckInputPending</code> is called then the user agent MUST run the following steps:</p>
-        <ol>
-          <li>If <code>SupportedEventTypes</code> is empty return false.</li>
-          <li>If inputType is not equal to an empty string and if inputType is not in <code>SupportedEventTypes</code>, return false.</li>
-          <li>Return true.</li>
-        </ol>
+        <p class="note">The specification for the isInputPending method is intentionally written in such a way that a user agent may return false for any reason. This is so user agents may include their own security measures while implementing this api and remain compliant with this specification.</p>
```

Comment:
We also need to call out that it is *not* ok to return true unless the user agent is sure that the event will be dispatched to the same Agent as the one that is querying isInputPending.

Also add a sentence explaining that for performance reasons, user agents may not always know to which Agent an input event will be dispatched without an expensive computation, and therefore the UA is allowed to avoid that computation and return false in such cases.